schemas/PreSource.yaml

```diff
@@ -0,0 +1,456 @@
+# This file defines the mapping between the columns in a single visit+detector
+# source table and their respective DPDD-style column names, as used by
+# `lsst.pipe.tasks.postprocess.TransformSourceTableTask`.
+# See the DPDD for more information about the output: https://lse-163.lsst.io
+funcs:
+    sourceId:
+        functor: Column
+        args: id
```

Comment:
The two lines above are the only difference between this file and `Source.yaml`.  I should be able to unify them again on [DM-34875](https://rubinobs.atlassian.net/browse/DM-34875), which makes `writeRecalibratedSourceTable` proudce an `ArrowAstropy` output just like `calibrateImage` does.